src/pilgrim/ui/screens/edit_entry_screen.py

```
@@ -742,15 +739,15 @@ def on_option_list_option_selected(self, event: OptionList.OptionSelected) -> No
-        photo_details = f"📷 {selected_photo.name}\n"
-        photo_details += f"🔗 {photo_hash}\n"
-        photo_details += f"📅 {selected_photo.addition_date}\n"
+        photo_details = f"Name: {selected_photo.name}\n"
+        photo_details += f"Hash: {photo_hash}\n"
+        photo_details += f"Date: {selected_photo.addition_date}\n"
         if selected_photo.caption:
-            photo_details += f"💬 {selected_photo.caption}\n"
-        photo_details += f"📁 {selected_photo.filepath}\n\n"
+            photo_details += f"Caption: {selected_photo.caption}\n"
+        else:
+            photo_details += f"Caption: No Caption\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)